.github/workflows/ci.yml

```diff
@@ -68,10 +68,22 @@ jobs:
             -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
             -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
             -v /tmp/nominatim-postgres:/var/lib/postgresql/12/main \
-            -p 8082:8080 \
+            -p 8003:8080 \
             nominatim &
           sleep 35
-          ./assert-non-empty-json "http://localhost:8082/search.php?q=avenue%20pasteur"
+          ./assert-non-empty-json "http://localhost:8003/search.php?q=avenue%20pasteur"
+
+      - name: Check container restart
+        working-directory: .github/workflows
```

Comment:
You're using the same volume with two running containers. Is this a supported usecase? Wouldn't it be wiser to stop the previous container?

Reply:
True, thanks for the update in your PR.